examples/shardingsphere-example-generator/src/main/resources/template/jdbc/java/TestQueryAssistedShardingEncryptAlgorithm.ftl

```diff
@@ -50,4 +50,9 @@ public final class TestQueryAssistedShardingEncryptAlgorithm implements QueryAss
     public String getType() {
         return "assistedTest";
     }
+
+    @Override
+    public void init(Properties props) {
```

Comment:
Thanks for your contribution, but this init method I think needs to initialize props, you can refer to the rest of the TestQueryAssistedShardingEncryptAlgorithm file

Reply:
ok，I see